ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/components/video/VideoComponentView.kt

```diff
@@ -57,7 +57,7 @@ internal fun VideoComponentView(
                 .clip(composeShape)
                 .applyIfNotNull(borderStyle) { border(it, composeShape).padding(it.width) },
         ) {
-            if (videoUrl == null && fallbackImageViewStyle != null) {
+            if (fallbackImageViewStyle != null) {
```

Comment:
This will cause the `ImageComponentView` to be rendered pretty much all the time (even though it'd be covered up by the video view once loaded), right? Would it be better from a resource management and accessibility perspective to remove this `ImageComponentView` from the view hierarchy when the video view is rendered in the `.let { }` block below?